lib/src/linux/kdialog_handler.dart

```diff
@@ -65,8 +65,6 @@ class KDialogHandler implements DialogHandler {
         return 'Media File (${DialogHandler.toCaseInsensitive("*.avi *.flv *.mkv *.mov *.mp4 *.mpeg *.webm *.wmv *.bmp *.gif *.jpeg *.jpg *.png")})';
       case FileType.video:
         return 'Video File (${DialogHandler.toCaseInsensitive("*.avi *.flv *.mkv *.mov *.mp4 *.mpeg *.webm *.wmv")})';
-      default:
```

Comment:
This is reported by https://dart.dev/tools/diagnostic-messages?utm_source=dartdev&utm_medium=redir&utm_id=diagcode&utm_content=unreachable_switch_default#unreachable_switch_default